File: TenMinsPy/day49.py
# Check if string contains at least 1 digit

import re

class day49:
    def __init__(self):
        s = input("enter a string with a number\n")
        p = r".*[0-9].*"
        if re.search(p, s):
            print('yeah man')
        else:
            print('nah man')

day49()

File: TenMinsJava/day49.java
// Check if string contains at least 1 digit
package TenMinsJava;

import java.util.Scanner;

public class day49 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.err.println("Enter string with a digit");
        String inp = scan.nextLine();
        if (inp.matches(".*[0-9].*")) {
            System.err.println("yes");
        }
        else {
            System.err.println("no");
        }
    }
}
